solidity/build/contracts/Address.json

```diff
@@ -0,0 +1,8066 @@
+{
```

Comment:
all the files in the `solidity/build` folder are generated by the build and should be removed from source control

Reply:
Done